aip-console-tools-cli/src/main/java/com/castsoftware/aip/console/tools/ApiIntegrationConfig.java

```diff
@@ -59,8 +59,8 @@ public UploadService chunkedUploadService(@Autowired RestApiService restApiServi
     }
 
     @Bean
-    public ApplicationService applicationService(@Autowired RestApiService restApiService, @Autowired JobsService jobsService) {
-        return new ApplicationServiceImpl(restApiService, jobsService);
+    public ApplicationService applicationService(@Autowired RestApiService restApiService, JobsService jobsService, @Autowired UploadService uploadService) {
```

Comment:
The `@Autowired` is missing in front of "JobsService"

That said, maybe you can remove all the `@Autowired` ? I think Spring should automatically put them without having the annotation but I'm not sure.

Reply:
yes